tests/helpers/healthevent.go

```diff
@@ -69,3 +80,134 @@ func SendHealthEventsToNodes(nodeNames []string, eventFilePath string) error {
 
 	return errors.Join(errs...)
 }
+
+const (
+	MONGODB_DATABASE_NAME   = "HealthEventsDatabase"
+	MONGODB_COLLECTION_NAME = "HealthEvents"
+)
+
+func getMongoCollection(ctx context.Context) (*mongo.Client, *mongo.Collection, error) {
```

Comment:
This is not correct. We should not connect to mongodb in tests, we should check outcomes -- node condition added, node event added, label added etc.